evap/results/views.py

```diff
@@ -141,8 +141,6 @@ def course_detail(request, semester_id, course_id):
 def user_can_see_text_answer(user, represented_users, text_answer, public_view=False):
     if public_view:
         return False
-    if text_answer.state not in COMMENT_STATES_REQUIRED_FOR_VISIBILITY:
-        return False
```

Comment:
this and the change in tools.py probably means that hidden comments will be visible to staff users when they are previewing the results page before publishing. is this intended?

Reply:
true. that wasn't intended. i changed it and added a test for that.